bunq/sdk/context/api_context.py

```diff
@@ -120,7 +96,7 @@ def _register_device(self, device_description,
         :rtype: None
         """
 
-        from bunq.sdk.model.device_server_internal import DeviceServerInternal
+        from bunq.sdk.model.core.device_server_internal import DeviceServerInternal
```

Comment:
why local import?

Reply:
We use local imports because of the way Python imports modules (https://stackabuse.com/python-circular-imports/).